sled-agent/Cargo.toml

```diff
@@ -127,14 +127,7 @@ name = "sled-agent"
 doc = false
 
 [features]
-image-standard = []
```

Comment:
To clarify, are all these features getting removed because they aren't used anywhere?

(If so, great, awesome to see these cleaned up)

Reply:
Yep, aiui these were all in sled-agent because the features had to be defined in _some_ package for the `cargo build` to work. But if you're not building sled-agent, you'll often get a failure to build because of features that don't exist in any of the selected packages.

I grepped for the feature names across the tree to identify which packages were actually using which feature names.